lib/schemes/oauth2.js

```diff
@@ -14,7 +24,10 @@ export default class Oauth2Scheme {
     this.req = auth.ctx.req
     this.name = options._name
 
-    this.options = Object.assign({}, DEFAULTS, options)
+    this.options = defu(options, DEFAULTS)
+    this.options.token.property = this.options.token_key || this.options.token.property
```

Comment:
Instead of silent backward compatibility, we can use a strict schema validator and migration path in docs.

Reply:
Ok! I'm working on it!

Reply:
I refactored all options of `oauth2` scheme. They are all camelCase now because all other options of the auth module are camelCase. I also refactored endpoint properties to `endpoints: { ... }`.
Commit https://github.com/nuxt-community/auth-module/pull/548/commits/b3cf4964de8bb2fee869a304e4d8e75564486946